sdk/eventgrid/azure-messaging-eventgrid/README.md

```diff
@@ -165,12 +165,6 @@ String sasToken = EventGridPublisherClient
     .generateSas("<your event grid endpoint>", new AzureKeyCredential("<key for the endpoint>"), expiration);
 ```
 
-### Use `BinaryData`
```

Comment:
Any reason for deleting this? We should keep this or link to a central location where BinaryData is documented.